nixos/modules/programs/dconf.nix

```diff
@@ -59,8 +58,9 @@ in
     # For dconf executable
     environment.systemPackages = [ pkgs.dconf ];
 
-    # Needed for unwrapped applications
-    environment.sessionVariables.GIO_EXTRA_MODULES = mkIf cfg.enable [ "${pkgs.dconf.lib}/lib/gio/modules" ];
+    environment.sessionVariables = {
+      # Needed for unwrapped applications
```

Comment:
Shouldn't we rather fix those or is the overhead so big, that we rather not wrap this at all?